public/index.html

```diff
@@ -41,3 +41,12 @@
     -->
   </body>
 </html>
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/emailjs-com@2.3.2/dist/email.min.js"
+></script>
+<script type="text/javascript">
+  (function () {
+    emailjs.init("jKUbAkhP5wr4KtFm6");
```

Comment:
Ojć, stwórz sobie proszę plik w główny katalogu, nazwij go .env , a w tym pliku stwórz EMAILJS_PASSWORD. aktualnie zakomitowalas swoje hasło do tego serwisu, co jest bardzo niebezpieczne tymbardziej, że repozytorium jest publiczne i każdy sobie może to zabrać i korzystać z Twojego konta w emailjs, a tam były progi aktywności, czyli ktoś może się zalogować na ciebie i przekroczyć próg darmowy i będzie trzeba płacić 😅